src/sql/workbench/parts/notebook/cellViews/output.component.html

```diff
@@ -6,6 +6,10 @@
 -->
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
 	<div style="flex: 0 0 auto; user-select: none;">
-		<div #output class="output-userselect" ></div>
+		<div #output class="output-userselect">
```

Comment:
Note: I had to keep this output div because putting a class on the component-host failed. An alternative is to try and fix CSS such that the parent div has more dynamic user-select behavior but for now this seems required.

This preserves existing select / select all behavior on non-mac machines at the expense of 1 extra, not really needed, div.